sdk/template/ci.yml

```diff
@@ -22,6 +22,13 @@ pr:
     - sdk/template/
     - sdk/eng/
 
+resources:
+  repositories:
+  - repository: SwiftPM-AzureTemplate
+    type: github
+    name: Azure/SwiftPM-AzureTemplate
+    endpoint: Azure
+
```

Comment:
Out of curiosity, what happens if a new SDK is added to the service directory but NOT to this resources section? Does it fail silently, or cause an error?

Reply:
When you go to trigger the release pipeline it would fail because it will try to checkout a repo that doesn't exist. To handle publishing packages to CocoaPods and not SPM, you can add a skip variable, and in those cases the absence of the resources section won't be a problem because in the final emitted pipeline it is not used.

Reply:
Perfect. The silent fail would be the scenario I feared.